### DISCUSSION TITLE
Secured branding

### DISCUSSION THREAD

**Discussion by salvo-uk** [PiranhaCMS/piranha.core#1406] [Q&A] [answered]
Hi,

Is it possible to secure Piranha CMS such that core branding (colours, fonts, heading styles etc) cannot be changed by regular content authors? i.e. So they can only be changed by developers or designated 'approvers' / admins.

Thanks
Salvo

**Comment by tidyui** [comment, marked as answer]
Hi there! Could you clarify as I'm not sure what you mean. Piranha CMS doesn't really include any settings for visual things which means that editors will never be able to edit them unless you course you've added them yourself into the manager.

If you'd like to be able to edit the branding of your application but **not** let the editors change it, one way to do it would be to.

1. Add all of your settings using `api.Params`. Here you can basically add any value as a database setting for your application.
2. Create a custom branding view inside the manager
3. Set permissions so only certain administrators can access the branding functionality.

Best regards

**Reply by tidyui** [reply]
Another option is also to add a `SiteType` which is like a `PageType` but you assign it to your site instead. This way you can add custom fields & regions that will be global to your site. After that you could just remove access to sites for the normal content editors and they won't be able to change it. This is solution also requires minimal coding from your part. Take a look at https://piranhacms.org/docs/content/sites for reference.

**Comment by salvo-uk** [comment]
Hi Håkan

Thank for the prompt response. I don’t think I need to clarify as you’ve answered my question perfectly well!

Thanks
Salvo

**Reply by tidyui** [reply]
Just updated my previous response with another option that requires less coding. If you think it answers your question, make sure to mark my response as "Answer" to your question!
